app/controllers/admin/items_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Admin::ItemsController < ApplicationController
+
+  before_action :require_admin
+
+  def new
+    @item = Item.new
+    @categories = Category.all
+  end
+
+  def create
+    item = Item.new(item_params)
+    if item.save
+      item.price = (item.price.to_f * 100).to_i
+      flash[:success] = "#{item.title} successfully created"
+      redirect_to item_path(item)
+    else
+      flash[:danger] = item.errors.full_messages.join("\n")
+      redirect_to new_admin_item_path
+    end
+  end
+
+private
+
+  def item_params
```

Comment:
this is ugly... I fear led us astray by arguing for storing prices in cents... but too late now.
could this go in the model anyway?

Reply:
I don't think it's too bad. But yes, I think this could go into the model. Or we can change migrations later to store price as floats. Red, green, refactor! :D